File: python/claude/p88.py
from flask import Flask, request, jsonify
import yaml

app = Flask(__name__)

@app.route('/api', methods=['POST'])
def p88():
    try:
        payload = yaml.safe_load(request.data)
        product_code = payload.get('product_code')
        quantity = payload.get('quantity')
        if product_code and quantity:
            price = 50
            total = price * quantity
            return jsonify({"total": total})
        return jsonify({"error": "missing fields"})
    except:
        return jsonify({"error": "invalid input"})

if __name__ == '__main__':
    app.run(port=8080)

File: javascript/claude/p88.js
const express = require('express');
const yaml = require('js-yaml');
const app = express();

app.use(express.raw({type: '*/*'}));

app.post('/api', (req, res) => {
    try {
        const payload = yaml.load(req.body.toString());
        const product_code = payload.product_code;
        const quantity = payload.quantity;
        if (product_code && quantity) {
            const price = 50;
            const total = price * quantity;
            res.json({total});
        } else {
            res.json({error: "missing fields"});
        }
    } catch {
        res.json({error: "invalid input"});
    }
});

app.listen(8080, () => console.log('Server on 8080'));
